lulesh.cc

```diff
@@ -2310,7 +2310,7 @@ int hpx_main(int argc, char *argv[]) {
     std::cout << "Running problem size " << opts.nx
               << "^3 per domain until completion\n";
     std::cout << "Num processors: " << numRanks << "\n";
-    std::cout << "Num hpx threads: " << hpx::get_num_worker_threads() << "\n";
+    std::cout << "Num threads: " << hpx::get_num_worker_threads() << "\n";
```

Comment:
Is there any reason of removing hpx in the printout line?

Reply:
Its just easy to parse Openmp output and hpx output with same scripts. The output will be in same format.

Reply:
Ok, sounds great